src/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.8)
+
+# Define the project name.
+project(numsim)
+
+# Specify the name of the executable (${PROJECT_NAME} which is
+# equal to what was set in the project() command).
+# Also specify the source files.
+add_executable(${PROJECT_NAME}
+   main.cpp
+   output_writer/write_paraview_output.cpp
+   settings_parser/settings.cpp
+)
+
+# Add the project directory to include directories,
+# to be able to include all project header files from anywhere
+target_include_directories(${PROJECT_NAME} PUBLIC ${PROJECT_SOURCE_DIR})
+
+# Search for the external package "VTK"
+
+find_package(VTK)
+
+if (NOT VTK_FOUND)
+  message("VTK was not found on the system, forunately I know where it is")
+  set(VTK_DIR /home/jonat/software/VTK-9.3.0.rc1/build)
```

Comment:
Fine for now but obviously needs to be changed eventually before we hand in the code